sdk/examples/client/block/03_block_custom_payload.rs

```diff
@@ -3,7 +3,12 @@
 
 //! This example sends a block with a custom payload.
 //!
-//! `cargo run --example block_custom_payload --release -- [NODE URL]`
+//! Rename `.env.example` to `.env` first, then run the command:
+//! ```sh
+//! cargo run --release --example block_custom_payload
+//! ```
+
+use std::env;
```

Comment:
I can see your point but it may be annoying to follow through with that. It makes the code itself (the important part IMO) less readable. Also the imports are separated so I don't think it's confusing. Also ... the example files should be similar to what they are most likely be looking in the user code. What will happen is that user will c&p the example code, and it would be more handy for them if they don't have to do many changes anymore (i.e. changing all the std/tokio/futures etc imports)

Reply:
Well it's very hard for me to ask you to revert now that you did it. But we already had this discussion and we said we'd keep the imports as SDK as we can.
I strongly disagree that scoping things makes code less readable, I don't see why that would be the case.
Most people's code won't use env anyway, this is just for example, so they'll have to remove it, having an import makes it one more thing to remove so a bit more annoying.
I guess I can live with it, it's fine, but please let's stick to we said, you're basically forcing our hand here.

Reply:
sorry, I must have forgotten that discussion. Why didn't you mention this already in the wallet examples PR? There I even fully import `std::env::var` :thinking: I don't mind changing it. Not  wanting to forcing anything. I am just saying that if done consistently this will mean fully qualified imports for other things except `env` as well, right?

Reply:
Sorry I may have worded things a bit strongly. I'm just saying we had exactly this in the past and changed it to scoping, approved and merged. And now you're bringing it back. We just need to be careful this doesn't happen too much, it's wasted time. If I didn't say anything in another PR is simply because I'm not bulletproof. Not sure what your last question means. But anyway, it's fine, we can keep it. I'm only saying this to be careful in the future.

Reply:
No offense taken. I myself am guilty of sometimes wording things too strongly. There's nothing here we fundamentally disagree, and I think there's pros and cons for both views. Since you're the lead in such situations you can just over-rule me and it's totally fine. 

The last question had those examples like the `offline-signing` examples in mind which need a bit more from std/tokio. I think fully qualiffyin/scoping the tokio/std imports doesn't help the example reader.

Reply:
removed (hopefully all) non-sdk imports